src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costOfAllDays = days * 40;
+
+  if (days < 3) {
+    return costOfAllDays;
+  }
+
+  if (days < 7) {
+    return costOfAllDays - 20;
+  }
+
+  return costOfAllDays - 50;
```

Comment:
Try to avoid magic numbers. All numbers that are unclear from the context should be put into constants with a name that is understandable in terms of content